.env.example

```diff
@@ -16,3 +16,14 @@ GC_NOTIFY_CALLBACK_BEARER_TOKEN=
 
 # Used when running against LocalStack
 LOCAL_AWS_ENDPOINT=http://localhost:4566
+
+# Local dev 
+# Temporary token templateID
+TEMPORAY_TOKEN_TEMPLATE_ID=7199f6c9-41de-4941-85be-6fa6919b05d9
+
+# A link to GC Nofify service like staging or production
+GC_NOTIFY_URL=https://staging.notification.cdssandbox.xyz
```

Comment:
We're not using notify staging for this. We just want a separate service in the production instance of Notify

Reply:
After  a chat with @Moro-Code  it seems that Notify staging is not guaranteed to up all time and free of bugs. 
For those reasons, i will be creating a production instance of Notify for staging.